DANS-KNAW/dd-dans-deposit-to-dataverse
# DD-147 Add documentation of service functionality

Fixes DD-147

# Description of changes
* Added te  plumbing to create a GitHub pages site for the documentation.
* Added documentation for the current functionality + notes about changes to be expected

This is a currently a draft, however the basic layout of the sections and subsections should make clear where the various parts of information are planned to go.

# How to test
1. Check out the PR locally
2. Make sure you have `mkdocs` and the DANS theme installed.
3. Run `mkdocs serve` from the root directory of the project.

# Related PRs 
(Add links)
* 

# Notify
@DANS-KNAW/dataversedans
